src/app/containers/IndexHeading/index.jsx

```diff
@@ -21,30 +27,57 @@ const StyledIndexHeading = styled(IndexHeading)`
   @media (min-width: ${GEL_GROUP_4_SCREEN_WIDTH_MIN}) {
     padding: ${GEL_SPACING_TRPL} 0 0;
   }
+`;
+
+const MostReadPaddingStyles = css`
+  padding: ${GEL_SPACING_DBL} 0 ${GEL_SPACING_TRPL};
+  @media (min-width: ${GEL_GROUP_3_SCREEN_WIDTH_MIN}) and (max-width: ${GEL_GROUP_3_SCREEN_WIDTH_MAX}) {
+    padding: ${GEL_SPACING_TRPL} 0 ${GEL_SPACING_DBL};
+  }
+  @media (min-width: ${GEL_GROUP_4_SCREEN_WIDTH_MIN}) {
+    padding: ${GEL_SPACING_TRPL} 0 ${GEL_SPACING};
+  }
+`;
 
+const paddingStyles = {
+  idx: IdxPaddingStyles,
+  mostRead: MostReadPaddingStyles,
+};
+
+const StyledIndexHeading = styled(IndexHeading)`
+  ${({ pageType }) => paddingStyles[pageType]}
+ 
   @media (min-width: ${GEL_GROUP_5_SCREEN_WIDTH_MIN}) {
     width: 100%; /* Needed for IE11 */
     margin: 0 auto;
     max-width: ${GEL_GROUP_5_SCREEN_WIDTH_MIN};
   }
 `;
 
-const IndexHeadingContainer = ({ children, ...props }) => {
+const IndexHeadingContainer = ({ children, pageType, ...props }) => {
   const { script, service, dir } = useContext(ServiceContext);
 
   return (
-    <StyledIndexHeading script={script} service={service} dir={dir} {...props}>
+    <StyledIndexHeading
+      script={script}
+      service={service}
+      dir={dir}
+      pageType={pageType}
+      {...props}
+    >
       {children}
     </StyledIndexHeading>
   );
 };
 
 IndexHeadingContainer.propTypes = {
   children: node,
+  pageType: PAGE_TYPES,
```

Comment:
Is there a reason why a definition for this is at the top of the page if `PAGE_TYPES` is only used for `propTypes`?

Reply:
It's fine either way. Just following the pattern we used in the `StoryPromo` container https://github.com/bbc/simorgh/blob/latest/src/app/containers/StoryPromo/index.jsx#L253